shinydashboard/lantern/server.R

```diff
@@ -134,8 +134,29 @@ function(input, output, session) { #nolint
      "validations_tab" = "Validations Page"
   )
 
+  output$resource_tab_popup <- renderUI({
+    if (show_resource_tab_popup()) {
+      actionButton("resource_popup", "How to use this page")
+    }
+  })
+
+  observeEvent(input$resource_popup, {
+    showModal(modalDialog(
+      title = "How to use this page...",
+      p("By default, the list of resources below contains the supported resources across all endpoints and FHIR versions. Clicking a resource in the left box selects it and moves it to the right box. Remove a resource from the list by clicking the resource in the right box.", style = "font-size:16px; margin-left:5px;"),
+      p("You may also change the FHIR Version or Developer filtering criteria to select the applicable supported resources from the default list.
+        Any selected resources at that point will be removed if no endpoints that pass the selected filtering criteria support the given resource.
```

Comment:
Maybe change it to "Any resources at that point will be removed from the list if no endpoints..."